### DISCUSSION TITLE
error while doing forge test

### DISCUSSION THREAD

**Discussion by guyzilberblum** [Cyfrin/security-and-auditing-full-course-s23#159] [Q&A]
i just started the Smart Contract Audits, Security, and DeFi FULL Course | Learn smart contract auditing and im doing forge test to the counter contract and its giving me this error 

```
Running 2 tests for test/Counter.t.sol:CounterTest
[FAIL. Reason: unknown selector `0x98296c54` for VmCalls; counterexample: calldata=0x5c7f60d70000000000000000000000000000000000000000000000000000000000000000 args=[0]] testFuzz_SetNumber(uint256) (runs: 0, μ: 0, ~: 0)
[FAIL. Reason: unknown selector `0x98296c54` for VmCalls] test_Increment() (gas: 31325)
Test result: FAILED. 0 passed; 2 failed; 0 skipped; finished in 2.50ms
 
Ran 1 test suites: 0 tests passed, 2 failed, 0 skipped (2 total tests)

Failing tests:
Encountered 2 failing tests in test/Counter.t.sol:CounterTest
[FAIL. Reason: unknown selector `0x98296c54` for VmCalls; counterexample: calldata=0x5c7f60d70000000000000000000000000000000000000000000000000000000000000000 args=[0]] testFuzz_SetNumber(uint256) (runs: 0, μ: 0, ~: 0)
[FAIL. Reason: unknown selector `0x98296c54` for VmCalls] test_Increment() (gas: 31325)
```

**Comment by EngrPips** [comment]
Hello @guyzilberblum, please run `foundryUP` in your terminal and re-run the test. Let me know if that works. If it didn't work, you shouldn't bother about the counter contract, and don't let that slow you down. Just continue the course with Patrick

**Comment by guyzilberblum** [comment]
Thanks I will try tomorrow I’m going to bed now

**Reply by EngrPips** [reply]
Alright, friend, sleep tight

**Comment by guyzilberblum** [comment]
I tried it it says commend not found any way I will continue the course

**Reply by EngrPips** [reply]
That is weird, actually, but continue as you have said, and maybe things will make more sense as you progress

**Comment by guyzilberblum** [comment]
Thanks again

**Reply by EngrPips** [reply]
Anytime friend
